common/buildings/00_capital_buildings.txt

```diff
@@ -40,14 +40,137 @@ building_colony_shelter = {
 
 	planet_modifier = {
 		planet_housing_add = 3
-		planet_amenities_add = 3
+		planet_amenities_add = 7
```

Comment:
My only issue with buildings is that some of them don't seem to have the asari modifier, can we check that again, this is one of the examples @maxim-a-k @mstonefield